### ISSUE TITLE
About BatchNorm in DagNN

### ISSUE THREAD

**Issue by xcyan** [vlfeat/matconvnet#331]
Can anyone give an example of constructing a batchnorm layer in DagNN framework?

I have tried:
net.addLayer('bn1', dagnn.BatchNorm(), {'x1'}, {'x2'}, {'filters', 'bias'}) 

but failed.

Thanks!

**Comment by vedaldi** [comment]
Hi, in the devel version you would need a third parameter for the moments:

net.addLayer('bn1', dagnn.BatchNorm(), {'x1'}, {'x2'}, {'filters', ‘bias’, ‘moments’})

but this might not be your case if you are using beta16. What is not working in your example?

**Comment by AruniRC** [comment]
Can we please have a working example of adding BatchNorm layers to a DAG, in addition to the examples showing bnorm layers added to a simple NN, which is then converted to a DAG ?
